hydra-node/src/Hydra/Chain/CardanoClient.hs

```diff
@@ -414,20 +416,22 @@ queryInShelleyBasedEraExpr ::
   LocalStateQueryExpr b p QueryInMode r IO a
 queryInShelleyBasedEraExpr sbe query =
   queryExpr (QueryInEra $ QueryInShelleyBasedEra sbe query)
-    >>= (liftIO . throwOnUnsupportedNtcVersion)
-    >>= (liftIO . throwOnEraMismatch)
+    >>= liftIO
+    . throwOnUnsupportedNtcVersion
+    >>= liftIO
+    . throwOnEraMismatch
```

Comment:
Why did you remove the parenthesis? The code is less readable now IMO. Specifically it is not so clear that the `throwOnXXX` functions are lifted and not the full expression after a liftIO (I'm not even 100% whether this is having the same semantics now)

Reply:
What parenthesis?